runtime/ftplugin/heex.vim

```diff
@@ -13,4 +13,16 @@ setlocal shiftwidth=2 softtabstop=2 expandtab
 setlocal comments=:<%!--
 setlocal commentstring=<%!--\ %s\ --%>
 
+" HTML: thanks to Johannes Zellner and Benji Fisher.
+if exists("loaded_matchit") && !exists("b:match_words")
+  let b:match_ignorecase = 1
+  let b:match_words = '<!--:-->,' ..
+	\	      '<:>,' ..
+	\	      '<\@<=[ou]l\>[^>]*\%(>\|$\):<\@<=li\>:<\@<=/[ou]l>,' ..
+	\	      '<\@<=dl\>[^>]*\%(>\|$\):<\@<=d[td]\>:<\@<=/dl>,' ..
+	\	      '<\@<=\([^/!][^ \t>]*\)[^>]*\%(>\|$\):<\@<=/\1>'
+  let b:html_set_match_words = 1
```

Comment:
This line can be removed.  It only serves a purpose for files sourcing the HTML ftplugin.